descriptors/mysql/debezium-mysql-1.9.0.Alpha2.json

```diff
@@ -27,7 +27,7 @@
           "format" : "int64",
           "title" : "Cluster ID",
           "description" : "A numeric ID of this database client, which must be unique across all currently-running database processes in the cluster. This connector joins the MySQL database cluster as another server (with this unique ID) so it can read the binlog. By default, a random number is generated between 5400 and 6400.",
-          "default" : 5405,
+          "default" : 6360,
```

Comment:
we should stop creating this with random number

Reply:
I'll log a follow-up Jira to address that.

Reply:
That random number is actually sourced from `MySqlConnectorConfig` because the `withDefault` method call uses the `randomServerId()` method call.  If we don't want this to be generated randomly, we would need to explicitly override that behavior in the catalog generator, right?

Reply:
I know. I am not sure if we really need the random number generator at all? Maybe we just drop that default, or hardcode a number, or move the random number generation somewhere else but not in the field declaration.